R/tools-def.R

```diff
@@ -23,6 +23,8 @@ NULL
 #'   behavior. Usually created by [tool_annotations()], where you can find a
 #'   description of the annotation properties recommended by the [Model Context
 #'   Protocol](https://modelcontextprotocol.io/introduction).
+#' @param .convert Should JSON inputs be automatically convert to their
```

Comment:
I'm not against `convert` but it might be worth considering `simplify` in line with the `simplify*` arguments of `jsonlite::fromJSON()`

Reply:
I think that ship has sailed since `extract_data()` already uses `convert`.